shadowsocks-csharp/Controller/Service/AvailabilityStatistics.cs

```diff
@@ -197,10 +197,8 @@ private void UpdateRecords()
                     _latencyRecords.TryGetValue(id, out latencyRecords);
                     StatisticsRecord record = new StatisticsRecord(id, inboundSpeedRecords, outboundSpeedRecords, latencyRecords);
                     /* duplicate server identifier */
```

Comment:
remove this comment as we no longer check duplicate here

Reply:
done